.travis.yml

```diff
@@ -45,7 +62,7 @@ jobs:
       script:
         # Update `AUTHORS.md`
         - export MAINTAINER_TOKEN=${GH_TOKEN}
-        - go get github.com/alxwr/maintainer
+        - go get github.com/myii/maintainer
```

Comment:
@alxwr No disrespect here.  I've since pushed my changes to the upstream repo and now only maintaining the few SaltStack Formulas specifics in my fork.  Furthermore, the long-term plan would be to start a repo under this org and then change all of these over.  It will be easier to establish that change if this link is this same across the board.